docs/section5/Rule5-4.md

```diff
@@ -1,8 +1,8 @@
 
-# Envelope - Rule 5-4
+# Envelope - Rule 5-2  
```

Comment:
I'm confused, why are we renaming 5-4 as 5-2?

Reply:
I think I mixed up the files for the PR somehow. Please ignore all changes for Rule5-4 here. RDX/XC/Rule5-4 should cover all the latest changes.

Reply:
@xingcx we need to make 3 changes to 5-2, 5-3, 5-4 and 5-5

1. Split each of these into 2 RDS, one for the Appendix A check and one for the thermal properties
2. The Appendix A RDS  will do the following checks (i) determine if a wall/roof/ceiling/floor using the surface tilt and 'adjacent_to' data elements. (ii) verify if the surface construction is specified with the 'surface_construction_input_option' set to 'layers' (iii) verify that a value for U, F, C has been provided for walls, slab on grade or below grade floor respectively.
3. The thermal properties RDS will do the following

1. Determine surface type: Determine if the surface is a wall/roof/ceiling/floor using the surface tilt and 'adjacent_to' data elements.
2. Determine Space Conditioning Category for each space
		a. For the second RDS which checks the U factor/R value, you would determine the space conditioning category based on the heating and cooling capacity within the 'HeatingVentilationAirConditioning' data group. It has data elements for sensible cool capacity and sensible heat capacity.
			i. Conditioned: 
						1. First we determine is a space is conditioned. For this the sensible cool capacity >= 3.4 Btu/h-ft2. or heating capacity >= values in 90.1-2019, table 3.2.
						2. If conditioned, we determine is the space is residential (we had identified residential space types for another RDS, please look that up)
			ii. If semiheated- an enclosed space within a building that is heated by a heating system
whose output capacity is greater than or equal to 3.4 Btu/h·ft2 of floor area but is
not a conditioned space. 
		b. Once the Space Conditioning Category  has been determined, the Surface Conditioning Category  needs to be identified for each surface of that space.
3. Determine Surface Conditioning Category for each Surface of a Space
		a. If space conditioning category = conditioned, 
			i. Check surface adjacency
						1. If Surface adjacent to ‘ambient’ (‘ambient’ will be renamed to ‘exterior’, so please use ‘exterior’ in the RDS) - these would fall under residential and non-residential columns in Table G3.4.1-8
						2. If Surface adjacent to unconditioned or semi-heated space - these would fall under semiheated column in Table G3.4.1-8
								a. To determine this, we would check if a surface is adjacent to ‘Interior’, then look at the ‘adjacent_space_id’ and check the Space Conditioning Category  of the adjacent space to determine if it is ‘Unconditioned’ or ‘Semi-heated’ based on the rules in #1 above.
						3. All other surfaces would be unregulated and must have properties in B-RMR = P-RMR=U-RMR. (This would include interior surfaces separating conditioned spaces.)
		b. If space conditioning category = semiheated
			i. Check surface adjacency
						1. Surface adjacent to ambient or conditioned space - use semiheated column in Table G3.4.1-8
						2. adjacent to unconditioned or semi-heated space - B-RMR = P-RMR=U-RMR. (This would include surfaces adjacent to unconditioned or semi-heated spaces.)

Reply:
@supriyagoel @JasonGlazer @mkarpman  It seems we also need to determine if a space if indirectly conditioned, which might not have any heating or cooling but still qualify as conditioned space and the logic could be pretty complex. Can we use **space.conditioning_type** to determine if a space is conditioned (including cooled space, heated space and indirectly conditioned space), semi-heated or unconditioned?  But right now it does not have indirectly conditioned type.

Reply:
I updated and changed this rule from 5-2 to 5-2-2 (check thermal properties) and will add a rule 5-2-1 for layer checks. Let me know if I missed anything @JasonGlazer @supriyagoel @mkarpman . A few notes I have:
- Applicability check is removed as the RDS will determine that. 
- Only P-RMR is referenced. I'm assuming P-RMR and U-RMR have the same constructions or other checks will handle that. 
- There might be cases where an indirectly conditioned space is adjacent to another indirectly conditioned space (which might not have been determined yet in the loop). Right now the RDS only uses directly conditioned spaces to determine the surface produce sum.
- Transfer rate > 3ACH is not currently included in the RDS to determine indirectly conditioned spaces. Do we need to add manual checks?
- There might be circular reference loop where the baseline construction is not correctly modeled hence affecting whether a space is indirectly conditioned hence affecting whether the construction is correct. But I guess it will raise error anyways so probably doesn't matter for our purpose.

Reply:
@xingcx 

- Your third bullet is a good point. I guess RCT should go through indirectly conditioned spaces identification loop several times until no new spaces are identified as indirectly conditioned during the cycle.
- I agree with the fourth bullet that this should be a manual check.
- I see your point in the fifth bullet. Will need to think about this one.
- For your